runtime/testy/studentAPI.py

```diff
@@ -7,6 +11,40 @@ def __init__(self, toManager, fromManager):
     self.fromManager = fromManager
     self.toManager = toManager
     self._createSensorMapping()
+    self._coroutines_running = set()
+
+  def run(self, fn, *args, **kwargs):
```

Comment:
Note that the counterpart `is_running` function is not currently part of this PR.